src/Package/MSBuild.UnGAC.Setup/MSBuild.UnGAC.Setup.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- The project MUST be net45. -->
+    <TargetFramework>net45</TargetFramework>
+    <!-- Set as an exe because this project publishes its own output. -->
+    <OutputType>Exe</OutputType>
+    <!-- Forcing the 'vsix' output to be json output. -->
+    <TargetVsixContainerName>MSBuild.UnGAC.Setup.json</TargetVsixContainerName>
+    <!-- VS Insertion -->
+    <VisualStudioInsertionComponent>Microsoft.Build.UnGAC.Setup</VisualStudioInsertionComponent>
```

Comment:
Can you make the exe name and namespace and folder and package name consistent? I'd say `Microsoft.Build.UnGAC`, but wouldn't argue if you want the `.Setup`.